src/js/scope.js

```diff
@@ -30,8 +31,8 @@
 		// JavaScript has some issues in its floating point implementation.
 		to = limit(parseFloat(to.toFixed(7)));
 
-		// Return false if handle can't move.
-		if ( to === scope_Locations[trigger] ) {
+		// Return false if handle can't move and ranges were not updated
+		if ( to === scope_Locations[trigger] && newScopeValue === scope_Values[trigger]) {
```

Comment:
I added a comment but it is not very good because it does not explain the underlying issue I was facing. Which I do not remember. I guess it handle the case were your update the option ranges and they are the same as before, thus you do not need to do anything more.